patches/CreatePBP.cmake

```diff
@@ -4,6 +4,7 @@
 # Args:
 # TARGET - defined by an add_executable call before calling create_pbp_file
 # TITLE - optional, string, target's name in PSP menu
+# BUILD_PRX - optional, generates and uses PRX file instead of ELF in EBOOT.PBP
```

Comment:
`ENC_PRX` needs to be documented too

Reply:
Regarding this. I think its kinda easier to maintain doc strings if they are next to its declaration. The cmake parser is very close to the top of the file so why not do this:

```cmake
set(oneValueArgs 
	TARGET          # defined by an add_executable call before calling create_pbp_file
	TITLE           # optional, string, target's name in PSP menu
	ICON_PATH       # optional, absolute path to .png file, 144x82
	BACKGROUND_PATH # optional, absolute path to .png file, 480x272
	PREVIEW_PATH    # optional, absolute path to .png file, 480x272
	)
set(options 
	BUILD_PRX # optional, generates and uses PRX file instead of ELF in EBOOT.PBP
	ENC_PRX   # optional, replaces PRX file with encrypted version. Only effective if you build PRX
	)
```
And it prompts additions or removal of features to be documented.

Reply:
Good idea, go for it